File: client/src/types/blogTypes.ts
export default interface Blog {
    id?: string;
    title: string;
    date: string;
    main_image: string;
    subheading: string;
    body: string;
    created?: string;
}

export interface Exhibition {
    id?: string;
    title: string;
    date: string;
    location: string;
    info: string;
    created?: string;
}

File: server/types/blogTypes.go
package types

type Blog struct {
	ID         string `json:"id"`
	Title      string `json:"title"`
	Date       string `json:"date"`
	MainImage  string `json:"main_image"`
	Subheading string `json:"subheading"`
	Body       string `json:"body"`
	Created    string `json:"created"`
}

type Exhibition struct {
	ID       string `json:"id"`
	Title    string `json:"title"`
	Date     string `json:"date"`
	Location string `json:"location"`
	Info     string `json:"info"`
	Created  string `json:"created"`
}
